R/labeller.r

```diff
@@ -426,7 +427,15 @@ labeller <- function(..., .rows = NULL, .cols = NULL,
   if (!is.null(keep.as.numeric)) {
     .Deprecated(old = "keep.as.numeric")
   }
+
   dots <- list(...)
+
+  if (length(dots) == 1) {
+    if (is.formula(dots[[1]]) || is.function(dots[[1]]) && !is_labeller(dots[[1]])) {
+      return(as_labeller(dots[[1]]))
+    }
+  }
+
```

Comment:
Can you elaborate on this? This seems orthogonal to the intent of the PR

Reply:
My bad, the chunk was meant to be deleted. I presume I was thinking about allowing to pass a lambda function directly as a `labeller()` argument back then, but that's way broader topic. From documetation, it is clear (to some extent) that unnamed arguments in `labeller()` is treated by `.default` labeller and `as_labeller()` should be used instead. Deleted in the upcoming commit to this PR branch.